Dockerfile

```diff
@@ -13,38 +15,46 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.13.15 as builder
+FROM golang:1.15.8 as builder
```

Comment:
golang should be 1.16.

Document was updated ;)
https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#minimum-go-version

Reply:
Fixed